src/client/containers/NextStepsPage/NextStepsPage.component.js

```diff
@@ -1,3 +1,145 @@
+                <img
+                  src={rediLogo}
+                  alt="redi-logo"
+                  width="50px"
+                  height="50px"
+                />
+              </NavLink>
+              <span className="image-name">Redi School</span>
+            </div>
+            &nbsp;
+            <div className="hyf-image-next-page">
+              <NavLink to="https://www.hackyourfuture.dk">
+                <img
+                  className="hyf-logo-next-page"
+                  src={hyfLogo}
+                  alt="hyf-logo"
+                  width="64px"
+                  height="50px"
+                />
+              </NavLink>
+              <span className="image-name">HackYourFuture School</span>
+            </div>
+          </div>
+          <p>
+            Or other website with some awsome{' '}
+            <strong>
+              Online <br /> courses like:
+            </strong>
+          </p>
+          <nav>
+            <ul className="footer-images">
+              <NavLink to="https://www.udacity.com/blog/category/free-courses">
+                <li>
+                  <img
+                    src={udacityImg}
+                    alt="udacity-img"
+                    width="75px"
+                    height="50px"
+                  />
+                </li>
+              </NavLink>
+              <NavLink to="https://www.coursera.org">
+                <li>
+                  <img
+                    src={courseraImg}
+                    alt="coursera-img"
+                    width="50px"
+                    height="50px"
+                  />
+                </li>
+              </NavLink>
+              <NavLink to="https://www.linkedin.com/">
+                <li>
+                  <img
+                    src={linkedinImg}
+                    alt="linkedin-img"
+                    width="165px"
+                    height="25px"
+                  />
+                </li>
+              </NavLink>
+              <NavLink to="https://www.lynda.com">
+                <li>
+                  <img
+                    src={lyndaImg}
+                    alt="lynda-img"
+                    width="50px"
+                    height="50px"
+                  />
+                </li>
+              </NavLink>
+              <NavLink to="https://careerfoundry.com">
+                <li>
+                  <img src={cfImg} alt="cf-img" width="50px" height="50px" />
+                </li>
+              </NavLink>
+              <NavLink to="https://www.udemy.com">
+                <li>
+                  <img
+                    src={udemyImg}
+                    alt="udemy-img"
+                    width="111px"
+                    height="44px"
+                  />
+                </li>
+              </NavLink>
+            </ul>
+          </nav>
+        </div>
+      </div>
+      <div className="circle-image">
+        <img className="c-image" src={roundImg} alt="round-img" />
```

Comment:
The Alt value is not a Classname so you can write "Guide IT"

Reply:
Thanks for review @dpfernandes .. its done as per your comment.. :)